### PR TITLE
Improvements for supporting UI Location

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: models/ssh_key.go [hunk 1]
```diff
@@ -740,6 +736,12 @@ func GetDeployKeyByRepo(keyID, repoID int64) (*DeployKey, error) {
 	return key, nil
 }
 
+// UpdateDeployKeyCols updates deploy key information only columns.
```
Comment (ethantkoenig) [inline models/ssh_key.go]: nit: `// UpdateDeployKeyCols updates deploy key information only in the specified columns`

File: models/issue_tracked_time.go [hunk 2]
```diff
@@ -24,7 +25,7 @@ type TrackedTime struct {
 
 // AfterLoad is invoked from XORM after setting the values of all fields of this object.
 func (t *TrackedTime) AfterLoad() {
-	t.Created = time.Unix(t.CreatedUnix, 0).Local()
+	t.Created = time.Unix(t.CreatedUnix, 0).In(setting.UILocation)
```
Comment (ethantkoenig) [inline models/issue_tracked_time.go:28]: Why have we left the `Created` field in the `TrackedTime` struct?

Comment (lunny) [reply models/issue_tracked_time.go:28]: It has one `json:"created"` tag here. I should invistigate where referenced it. Maybe send a PR later.

File: models/ssh_key.go (hunk 1 above)
Comment (lunny) [reply models/ssh_key.go]: done.

**Comment by lunny** [comment]
should fix #3178

**Comment by lafriks** [comment]
LGTM

**Comment by ethantkoenig** [comment]
LGTM

**Pull request merged**